src/index.ts

```diff
@@ -85,7 +85,7 @@ export class BondPlatform {
       return;
     }
 
-    const accessory = new Accessory(device.location + ' ' + device.type, hap.UUIDGen.generate(device.id.toString()));
+    const accessory = new Accessory(device.location + ' ' + device.name, hap.UUIDGen.generate(device.id.toString()));
```

Comment:
While you're in here, can you change to use string interpolation as well?
```
`${device.location} ${device.name}`
```

Reply:
Sure thing